n_helpers.c

```diff
@@ -376,7 +423,7 @@ const char * NoteBinaryTransmit(uint8_t * data, size_t dataLen, size_t bufLen, s
         // Issue a "card.binary.put"
         J *req = NoteNewRequest("card.binary.put");
         if (req) {
-            JAddIntToObject(req, "cobs", encLen);
+            JAddIntToObject(req, "cobs", outLen);
```

Comment:
I don't understand this rename.

`encLen` (encoded length) seems more expressive in the context of this function. If it was renamed to match the parameters of `NoteBinaryDecode`, then should it not be `inLen`?

Reply:
It's an in/out parameter now. On input, it's the length of the output buffer, so encLen didn't feel like a great name anymore. I felt outLen kind of captured the semantics of both input and output: on input, it's the length of the output buffer, and on output, it's the length of the actual data in the output buffer.